Tasks/HelmDeployV0/task.json

```diff
@@ -140,7 +140,7 @@
             "label": "Azure subscription for Container Registry",
             "helpMarkDown": "Select an Azure subscription, which has your Azure Container Registry.",
             "defaultValue": "",
-            "required": false,
+            "required": true,
```

Comment:
This could cause back-compat issues

Reply:
The command doesn't work if these fields are not given (The task fails). So, I don't think anyone would be using it without giving these inputs.